test/integration/routes.test.js

```diff
@@ -116,5 +116,37 @@ describe('routes', function () {
       expect(res.status).to.equal(409)
       expect(res.body).deep.equal(expectedResult)
     })
+
+    test('get member by alias', async function () {
+      const expectedResult = [
+        {
+          address: '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY',
+          alias: 'ALICE_STASH_UPDATE',
+        },
+      ]
+
+      const res = await getMembersByAliasOrAddressRoute(app, 'ALICE_STASH_UPDATE', authToken)
+
+      expect(res.status).to.equal(200)
+      expect(res.body).deep.equal(expectedResult)
+    })
+
+    test('get member by address', async function () {
+      const expectedResult = [
+        {
+          address: '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY',
+          alias: 'ALICE_STASH_UPDATE',
+        },
+      ]
+
+      const res = await getMembersByAliasOrAddressRoute(
+        app,
+        '5GNJqTPyNqANBkUVMN1LPPrxXnFouWXoe2wNSmmEoLctxiZY',
+        authToken
+      )
+
+      expect(res.status).to.equal(200)
+      expect(res.body).deep.equal(expectedResult)
+    })
```

Comment:
These tests rely on previous tests completeing successfully. When you merge in `main` these tests will likely break so you need to do that and then update these